liquidator/test/Liquidator.js

```diff
@@ -308,17 +308,4 @@ contract("Liquidator.js", function(accounts) {
     // The liquidation should have gone through.
     assert.equal((await emp.getLiquidations(sponsor1)).length, 1);
   });
-
-  it.only("set up positions", async function() {
```

Comment:
@chrismaree Is it OK to delete this test? It failed when used in conjunction with the other tests (i.e. removing the `.only()` part.